pkg/watcher/watcher.go

```diff
@@ -17,20 +16,6 @@ type Watcher struct {
 	metrics *WatcherMetrics       // Prometheus metrics for a validation key
 }
 
-// CleanupValidationKey sanitizies the given validation key.
-func cleanupValidationKey(key string) string {
```

Comment:
Not sure why I'm not using this, does the API handle this?

I initially had this as in my use-case, we tend to not prefix with 0x in our infra, and some settings left and right on existing eth validators accept both format, but I wanted the calls to the API to be as explicit as possible as it could be ambiguous (technically you could have a full hex address with only numbers, and you wouldn't differentiate it easily from a validator index).

Reply:
the linter complained it is not used and thus I removed it.

API works with both!

Reply:
👌